### PR TITLE
Add default currency for DRF field #475

### PR DESCRIPTION
**Pull request by butorov**
What do you think? :)
If the approach is ok, I may add some description into the documentation.

### CODE REVIEW

File: tests/contrib/test_django_rest_framework.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 import pytest
 
+import six
+from djmoney.contrib.django_rest_framework import MoneyField
```
Comment (Stranger6667) [inline tests/contrib/test_django_rest_framework.py]: Could you make this import local to avoid failing test build?

**Comment by Stranger6667** [comment]
Yep, I like it, could you, pls, fix the note above? then we can merge it

**Comment by Stranger6667** [comment]
Could you rebase, please?

**Comment by butorov** [comment]
Sure! Done.

**Pull request merged**